### PR TITLE
Follow git best practices in documents

### PR DESCRIPTION
**Pull request by mira01**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: MAINTENANCE.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 * Create pull request with the latest commit with bumped versions
 * Ask for merge of pull request. Once it is merged:
 * Checkout latest master tag it vX.Y.Z
-* Push the tag to master branch (`git push --tags`)
+* Push the tag to master branch (eg. `git push origin v.X.Y.Z`)
```
Comment (pcerny) [inline MAINTENANCE.md:9]: - `origin` is misleading as it is dependent on remote setup. I personally have main repo marked as `upstream` and my fork as `origin`.
- `master branch` is also not correct

What about the following formulation:

Push the tag to the `gooddata/gooddata-python-sdk` repository (e.g. `git push <remote> v.X.Y.Z` where remote is bind to the `gooddata/gooddata-python-sdk` )

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: MAINTENANCE.md (hunk 1 above)
Comment (mira01) [reply MAINTENANCE.md]: You are right!

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR66

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/59/ : SUCCESS in 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/59/ : SUCCESS in 1m 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/59/ : SUCCESS in 19s

**Pull request merged**